packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/endpoint_client/endpoint_info_store_manager.dart

```diff
@@ -9,25 +9,18 @@ import 'package:amplify_analytics_pinpoint_dart/src/impl/flutter_provider_interf
 import 'package:amplify_core/amplify_core.dart';
 import 'package:amplify_secure_storage_dart/amplify_secure_storage_dart.dart';
 
-/// Storage scope for [EndpointInfoStoreManager].
-/// Used to determine where Endpoint info is stored.
-enum EndpointStorageScope {
-  ///
-  analyticsPinpoint,
-
-  ///
-  pushNotifications,
-}
-
 /// {@template amplify_analytics_pinpoint_dart.endpoint_info_store_manager}
 /// Manages and provides Pinpoint Endpoint id and global fields.
 /// {@endtemplate}
 class EndpointInfoStoreManager {
   /// {@macro amplify_analytics_pinpoint_dart.endpoint_info_store_manager}
   EndpointInfoStoreManager({
-    required SecureStorageInterface store,
+    SecureStorageInterface? store,
```

Comment:
It is nullable because it is an optional parameter.  If it is not set a SecureStorageWorker is created instead.  This is consistent with how Analytics worked beforehand except that the default value is now defined in `EndpointInfoStoreManager` instead of the top level Analytics class.

Reply:
Why is it optional? I don't see a use of this constructor where a value is not provided. Is there a use case for that?